app/routes/endemics/timings.js

```diff
@@ -0,0 +1,42 @@
+const Joi = require('joi')
```

Comment:
I believe this screen doesn't need validation because there are only two buttons without any kind of input element.